### PR TITLE
Remove plugins

### LINKED ISSUE #371: Move plugins to own repo
See https://github.com/eWaterCycle/ewatercycle/issues/359

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #371

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (BSchilperoort) [inline .github/workflows/ci.yml:1]: Nice that you changed this to micromamba :rocket:

File: README.md
Comment (BSchilperoort) [inline README.md:1]: The readme says "More examples can be found in the [documentation](https://ewatercycle.readthedocs.io/).", however I think that's not the case anymore right?

Comment (BSchilperoort) [inline README.md:1]: "Exposes [simple interface](https://ewatercycle.readthedocs.io/en/latest/examples/ewatercycle_api_notebook.html) to quickly get up and running"

link is broken

**Review by BSchilperoort — APPROVED** [review]
Nice work! All plugin things seem to be removed. I did notice some outdated/broken links in the README.

File: .github/workflows/ci.yml
Comment (sverhoeven) [reply .github/workflows/ci.yml:1]: Yeah but somtimes the action fails with shared library errors

Comment (BSchilperoort) [reply .github/workflows/ci.yml:1]: Odd, I have not encountered that in Zampy's CI where we have a micromamba workflow.

**Pull request merged**